theme/landing.html

```diff
@@ -1,259 +1,686 @@
+              conciseness, maintainability, and deployability. When you choose
+              Keras, your codebase is smaller, more readable, easier to iterate
+              on.
+            </p>
+
+            <div class="hero__buttons">
+              <a class="button__square button__square--white" href="/api/">API DOCS</a>
+              <a class="button__square button__square--white" href="/guides/">GUIDES</a>
+              <a class="button__square button__square--white" href="/examples/">EXAMPLES</a>
+            </div>
+          </div>
+          <img class="hero__image" src="./img/k-graphic.png" />
+        </div>
+      </div>
 
-<head>
-
-  <meta charset="utf-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
-  <meta name="description" content="Keras documentation">
-  <meta name="author" content="Keras Team">
-
-  <title>Keras: Deep Learning for humans</title>
-
-  <!-- Bootstrap core CSS -->
-  <link href="css/bootstrap.min.css" rel="stylesheet">
-
-  <!-- Custom fonts for this template -->
-  <link href="https://fonts.googleapis.com/css?family=Open+Sans:wght@300;400;600;800&display=swap" rel="stylesheet">
-
-  <!-- Custom styles for this template -->
-  <link href="css/landing.css" rel="stylesheet">
-
-  <!-- Google Tag Manager -->
-  <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
-  new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
-  j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
-  'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
-  })(window,document,'script','dataLayer','GTM-5DNGF4N');
-  </script>
-  <script>
-  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
-  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
-  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
-  })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
-  ga('create', 'UA-175165319-128', 'auto');
-  ga('send', 'pageview');
-  </script>
-  <!-- End Google Tag Manager -->
-
-</head>
-
-<body>
-  <!-- Google Tag Manager (noscript) -->
-  <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-5DNGF4N"
-  height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
-  <!-- End Google Tag Manager (noscript) -->
-
-  <!-- Masthead -->
-  <header class="masthead smooth-white-bg text-center">
+      <div class="stack--container">
+        <div class="explore stacked__item">
+          <div class="explore__wrapper">
+            <div class="explore__column">
+              <div class="code__container">
+                <button class="code__copy--button">
+                  <i class="icon--copy"></i>
+                </button>
+                <pre>
+                  <code class="language-python">
+    inputs = keras.Input(shape=(32, 32, 3))
```

Comment:
We should remove the tab from these pre blocks so they copy without extra spaces. Instead just style the code card padding to get whatever left alignment we want.